client/pwa/src/db.ts

```diff
@@ -40,10 +41,33 @@ interface Whoami {
   is_subscriber: boolean;
 }
 
+export enum ContentStatusPhase {
```

Comment:
nit: Prefer enum identifiers in CAPS e.g INITIAL, IDLE

Reply:
Good point, fixed in 6b7710e37.